pages/whats-new/releases.html

```diff
@@ -16,16 +16,21 @@
 
 <p class="site-text-intro">Here you’ll find our release notes — summaries of bug fixes, new features, and other updates introduced in each release.</p>
 
-Have suggestions for a new feature or bug fix? [Open an issue](https://github.com/uswds/uswds/issues/new) in our repo.
+<p>Have suggestions for a new feature or bug fix? <a href="https://github.com/uswds/uswds/issues/new">Open an issue</a> in our repo.</p>
 
 {% for release in site.data.releases %}
 
-## Version {{ release.name }}
+<h2 id="version-{{ release.name | replace: ".", "" | slugify }}">Version {{ release.name }}</h2>
 
 <p class="site-subheading">{{ release.published_at | date: "%B %d, %Y" }}</p>
 
 {% assign id_replace = 'id="v%-' | replace: '%', release.name %}
-{{ release.body | markdownify | replace: 'id="', id_replace | remove_relative_links }}
-
+<div class="highlighter-rouge">
```

Comment:
Export release notes as plain text to avoid illogical heading flow

Reply:
Is this display of [release notes](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/dw-add-pa11y/about/releases/) intentional (gray box w/ markdown)?

**Example**
![image](https://user-images.githubusercontent.com/3385219/118682018-ad5e2280-b7c5-11eb-9f75-7d88620008a0.png)

Reply:
Yes, that's intentional — this prevents the headings of the original notes from affecting the page